apps/Gruntfile.js

```diff
@@ -443,37 +443,38 @@ describe('entry tests', () => {
   }));
 
   var codeStudioEntries = {
-    'blockly':                      './src/sites/studio/pages/blockly.js',
```

Comment:
Commenting because this is a confusing diff: updated the spacing on all of the entry points to match, then added a new entry point at the end

Reply:
https://github.com/code-dot-org/code-dot-org/pull/24880/files?w=1 hides the whitespace differences :)